apps/fabric-website-resources/README.md

```diff
@@ -1,9 +1,9 @@
 # @uifabric/fabric-website-resources
 
-**Local demo app for office-ui-fabric-react controls**
+**Local demo app for @fluentui/react controls**
 
-This package contains the demo app that runs when you do `yarn start` from the root of the Fabric repo or from the `office-ui-fabric-react` package.
+This package contains the demo app that runs when you do `yarn start` from the root of the Fluent UI repo or from the `@fluentui/react` package.
```

Comment:
Revert this one for now since technically start is still running from OUFR.
```suggestion
This package contains the demo app that runs when you do `yarn start` from the root of the Fluent UI repo or from the `office-ui-fabric-react` package.
```